themes/night/theme.lua

```diff
@@ -0,0 +1,112 @@
+theme.bg_urgent = "#df5b61"
+theme.bg_minimize = "#000a0e"
+theme.bg_systray = theme.bg_normal
+
+theme.fg_normal = "#d9d7d6"
+theme.fg_focus = "#6791c9"
+theme.fg_urgent = "#d9d7d6"
+theme.fg_minimize = "#d9d7d6"
+
+theme.useless_gap = dpi(2)
+theme.border_width = dpi(2)
+theme.border_color_normal = "#061115"
+theme.border_color_active = "#000a0e"
+theme.border_color_marked = "#000a0e"
+
+-- Generate taglist squares:
+local taglist_square_size = dpi(0)
+theme.taglist_squares_sel = theme_assets.taglist_squares_sel(taglist_square_size, theme.fg_normal)
+theme.taglist_squares_unsel = theme_assets.taglist_squares_unsel(taglist_square_size, theme.fg_normal)
+
+-- Variables set for theming the menu:
+theme.menu_submenu_icon = themes_path .. "night/submenu.png"
+theme.menu_height = dpi(15)
+theme.menu_width = dpi(100)
+
+-- Define the image to load
+theme.titlebar_close_button_normal = themes_path .. "night/titlebar/close_normal.png"
+theme.titlebar_close_button_focus = themes_path .. "night/titlebar/close_focus.png"
+
+theme.titlebar_minimize_button_normal = themes_path .. "night/titlebar/minimize_normal.png"
+theme.titlebar_minimize_button_focus = themes_path .. "night/titlebar/minimize_focus.png"
+
+theme.titlebar_ontop_button_normal_inactive = themes_path .. "night/titlebar/ontop_normal_inactive.png"
+theme.titlebar_ontop_button_focus_inactive = themes_path .. "night/titlebar/ontop_focus_inactive.png"
+theme.titlebar_ontop_button_normal_active = themes_path .. "night/titlebar/ontop_normal_active.png"
+theme.titlebar_ontop_button_focus_active = themes_path .. "night/titlebar/ontop_focus_active.png"
+
+theme.titlebar_sticky_button_normal_inactive = themes_path .. "night/titlebar/sticky_normal_inactive.png"
+theme.titlebar_sticky_button_focus_inactive = themes_path .. "night/titlebar/sticky_focus_inactive.png"
+theme.titlebar_sticky_button_normal_active = themes_path .. "night/titlebar/sticky_normal_active.png"
+theme.titlebar_sticky_button_focus_active = themes_path .. "night/titlebar/sticky_focus_active.png"
+
+theme.titlebar_floating_button_normal_inactive = themes_path .. "night/titlebar/floating_normal_inactive.png"
+theme.titlebar_floating_button_focus_inactive = themes_path .. "night/titlebar/floating_focus_inactive.png"
+theme.titlebar_floating_button_normal_active = themes_path .. "night/titlebar/floating_normal_active.png"
+theme.titlebar_floating_button_focus_active = themes_path .. "night/titlebar/floating_focus_active.png"
+
+theme.titlebar_maximized_button_normal_inactive = themes_path .. "night/titlebar/maximized_normal_inactive.png"
+theme.titlebar_maximized_button_focus_inactive = themes_path .. "night/titlebar/maximized_focus_inactive.png"
+theme.titlebar_maximized_button_normal_active = themes_path .. "night/titlebar/maximized_normal_active.png"
+theme.titlebar_maximized_button_focus_active = themes_path .. "night/titlebar/maximized_focus_active.png"
+
+theme.wallpaper = themes_path .. "night/background.png"
```

Comment:
It could be cool to use the new wallpaper API to generate the image rather than having yet another 1920x1080 PNG with AWESOME in capital letters :thinking:

Reply:
did u mean, i should generate an images with `theme_assets.wallpaper` rather than use an png images?

Reply:
~~Yeah, could be a solution.~~

Hm, with a second though, it adds some complications I'm too lazy to think about, so let's keep the same behavior as other theme :shrug:

Reply:
so just keep this 1920x1080 PNG images as wallpaper?

Reply:
Yup

Reply:
i think since you already started fixing the theme you could address this problem as well